gap/prop.gi

```diff
@@ -469,3 +469,27 @@ function(g)
   return true;
 
 end);
+
+
```

Comment:
Between functions there should be a blank line, then a line with just `#`, then another blank line.